components/layouts/AuthProvider.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
+import { Provider, useDispatch, useSelector } from 'react-redux'
+import Cookies from 'universal-cookie'
+import { isEmpty } from 'lodash'
+
+import { getMe } from '../../redux/slices/authSlice'
+import store from '../../store'
+
+const AuthProvider = ({ children }) => {
+  const [accessCookies, setAccessCookies] = useState('')
+  const dispatch = useDispatch()
+  const { user } = useSelector(state => state?.auth)
+
+  const cookies = new Cookies()
+
+  useEffect(() => {
```

Comment:
change to 

const token = cookies.get('token')

  useEffect(() => {
    if (isEmpty(user) && token) {
      dispatch(getMe())
    }
  }, [token])